UnleashedRecomp/main.cpp

```diff
@@ -178,6 +181,12 @@ int main(int argc, char *argv[])
         }
     }
 
+    if (!bypassCPURequirements && !os::host::IsCapableCPU())
+    {
+        SDL_ShowSimpleMessageBox(SDL_MESSAGEBOX_ERROR, GameWindow::GetTitle(), Localise("System_UnsupportedCPU").c_str(), GameWindow::s_pWindow);
+        std::_Exit(1);
+    }
+
     Config::Load();
```

Comment:
The localization here won't work because the config is not loaded before this check is made. As such it will be always in the default initialized value of the language setting.